sdk/python/pulumi_kubernetes/core/v1/Pod.py

```diff
@@ -30,6 +31,24 @@ def __init__(self, __name__, __opts__=None, metadata=None, spec=None, status=Non
             __props__,
             __opts__)
 
+    def get(self, __name__: str,
+            identifier: pulumi.Input[str],
+            __opts__: Optional[pulumi.ResourceOptions] = None
+            ):
+        """
+        Get the state of an existing `Pod` resource, as identified by `identifier`.
+        Typically this identifier  is of the form <namespace>/<name>; if <namespace> is omitted, then (per
+        Kubernetes convention) the identifier becomes default/<name>.
+       
+        Pulumi will keep track of this resource using `name` as the Pulumi ID.
+    
+        :param __name__: _Unique_ name used to register this resource with Pulumi.
+        :param identifier: An ID for the Kubernetes resource to retrieve. Takes the form
+        <namespace>/<name> or <name>.
+        :param __opts__: Uniquely specifies a CustomResource to select.
+        """
+        return Pod(__name__, __opts__, )
```

Comment:
@hausdorff I can't figure out where to use `identifier`. It looks like the `pulumi/pulumi` nodejs SDK includes an `id` field in `ResourceOptions`, but I don't see a corresponding field for the python SDK. Would this also require changes to `pulumi/pulumi`?